static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -172,240 +180,188 @@ class Content extends Component<Props, State> {
+          onAddressToggle: handleToggleAddresses,
+          image: findImageForAddress(frame.instructionAddr, frame.addrMode),
+          maxLengthOfRelativeAddress: maxLengthOfAllRelativeAddresses,
+          registers: {}, // TODO: Fix registers
+          isFrameAfterLastNonApp: isFrameAfterLastNonApp(),
+          includeSystemFrames,
+          onFunctionNameToggle: handleToggleFunctionName,
+          onShowFramesToggle: (e: React.MouseEvent<HTMLElement>) => {
+            handleToggleFrames(e, frameIndex);
+          },
+          isSubFrame: hiddenFrameIndices.includes(frameIndex),
+          isShowFramesToggleExpanded: toggleFrameMap[frameIndex],
+          showCompleteFunctionName,
+          isHoverPreviewed,
+          frameMeta: meta?.frames?.[frameIndex],
+          registersMeta: meta?.registers,
+          debugFrames,
+          isANR,
+          threadId,
+          lockAddress,
+          hiddenFrameCount: frameCountMap[frameIndex],
+        };
+
+        nRepeats = 0;
+
+        if (frameIndex === firstFrameOmitted) {
+          return (
+            <Fragment key={frameIndex}>
+              <DeprecatedLine {...frameProps} />
+              {renderOmittedFrames(firstFrameOmitted, lastFrameOmitted)}
+            </Fragment>
+          );
+        }
+
+        return <DeprecatedLine key={frameIndex} {...frameProps} />;
       }
 
       if (!repeatedFrame) {
         nRepeats = 0;
       }
 
-      if (frameIdx === firstFrameOmitted) {
-        frames.push(this.renderOmittedFrames(firstFrameOmitted, lastFrameOmitted));
+      if (frameIndex !== firstFrameOmitted) {
+        return null;
       }
-    });
-
-    if (frames.length > 0 && data.registers) {
-      const lastFrame = frames.length - 1;
-      frames[lastFrame] = cloneElement(frames[lastFrame], {
-        registers: data.registers,
-      });
-    }
 
-    if (defined(maxDepth)) {
-      frames = frames.slice(-maxDepth);
-    }
+      return renderOmittedFrames(firstFrameOmitted, lastFrameOmitted);
+    })
+    .filter(frame => !!frame) as React.ReactElement[];
 
-    if (newestFirst) {
-      frames.reverse();
-    }
-
-    const className = this.getClassName();
-    const platformIcon = stackTracePlatformIcon(platform, data.frames ?? []);
+  if (convertedFrames.length > 0 && registers) {
+    const lastFrame = convertedFrames.length - 1;
+    convertedFrames[lastFrame] = cloneElement(convertedFrames[lastFrame], {
+      registers,
+    });
+  }
 
-    return (
-      <Wrapper className={className} data-test-id="stack-trace-content">
-        {!hideIcon && <StacktracePlatformIcon platform={platformIcon} />}
-        <GuideAnchor target="stack_trace">
-          <StyledList data-test-id="frames">{frames}</StyledList>
-        </GuideAnchor>
-      </Wrapper>
-    );
+  if (defined(maxDepth)) {
+    convertedFrames = convertedFrames.slice(-maxDepth);
   }
-}
 
-export default withOrganization(Content);
+  const className = `traceback ${
```

Comment:
I like that this is much simpler now! One thing though, looks like it previously included the `className` from props in here as well